framework/kms-keyring.md

```diff
@@ -94,46 +99,35 @@ and MAY return a KMS Client that can make the following API calls in the given A
 If the client supplier cannot supply a client for the requested region,
 it MUST communicate that fact to the KMS keyring.
 
-### Key IDs
-
-Key IDs is a list of strings identifying KMS CMKs, in ARN format.
-This list identifies the CMKs to be used for data key encryption and decryption with this keyring.
+### Key Names
 
-Each Key ID MUST be one of the following:
+The key name identifies the [AWS KMS customer master keys (CMKs)](https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#key-id)
+that the AWS KMS keyring uses for data key encryption and decryption.
 
-- A CMK [alias](https://docs.aws.amazon.com/kms/latest/developerguide/programming-aliases.html) (e.g. "alias/MyCryptoKey")
-- A well-formed key ARN (e.g. arn:aws:kms:us-east-1:999999999999:key/01234567-89ab-cdef-fedc-ba9876543210)
-- A well-formed alias ARN (e.g. arn:aws:kms:us-east-1:999999999999:alias/MyCryptoKey)
+The AWS KMS keyring accepts all [key identifiers accepted by AWS KMS](https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#key-id) as key names.
```

Comment:
The actor here feels a bit muddled. Maybe:
```suggestion
The AWS KMS keyring accepts any value for a key name that is [accepted by AWS KMS as  a key identifer](https://docs.aws.amazon.com/kms/latest/developerguide/concepts.html#key-id).
```

There's also still an implied statement here that the keyring *does not* accept values that *are not* accepted by AWS KMS, which is not something that we can guarantee...I'm not sure what the right answer is for that.

Reply:
I like the change. Updating to it locally.

I think it might make sense to add a statement along the lines of "Key names are passed as-is to AWS KMS, without additional validation" or something else to be very clear.